src/PolyhedralGeometry/PolyhedralComplex/constructors.jl

```diff
@@ -0,0 +1,86 @@
+###############################################################################
+###############################################################################
+### Definition and constructors
+###############################################################################
+###############################################################################
+
+struct PolyhedralComplex
+    pm_complex::Polymake.BigObject
+    function PolyhedralComplex(pm::Polymake.BigObject)
+        return new(pm)
+    end
+end
+
+pm_object(pc::PolyhedralComplex) = pc.pm_complex
+
+
+@doc Markdown.doc"""
+    PolyhedralComplex(Polytopes, VR, far_vertices, Lineality)
```

Comment:
Is `Lineality` the same as the argument `L` in the actual function declaration? This argument is not mentioned in the description of the arguments either.